### ISSUE TITLE
Adjusting Global Settings in Free CAD

### ISSUE THREAD

**Issue by HeavyJohnny** [jlecomte/ascom-flat-panel#5]
Thank you so much for making this project public. I purchased everything to build this project and I sat down to print out the housing using the free cad file you provided. However, when resizing the unit to fit my esprit 120 using the global settings (OTA Outside diameter 175mm, and OTA thickness 7mm) I get an error on Pad009 Sketch018.  Recompute Failed. I am not a Free CAD user I use OnShape for all my projects so I have no clue how to correct it or even how to drill down to look at the error. Is there another parameter I need to change in Globals? Thanks again

**Comment by jlecomte** [comment]
If you can wait a few more months, I’ll have an entirely new version that will resolve this issue and bring several improvements. To find out when it’s available, subscribe to my YouTube channel. I’ll probably make a detailed video about it.

**Comment by HeavyJohnny** [comment]
I have been subscribed to you channel for a while now, I enjoy it, and I cant wait to see the improvements! In the next video I would love to see you build it too. It would be awesome to follow along with you. Thanks again. Also, I was able to print it despite the errors. It turned out great too.

**Comment by jlecomte** [comment]
Yes, absolutely, for the new and improved version, I will show how it is assembled and how I cleanly cut the diffuser and the LGP. Hopefully, this will happen before the end of this year, if all goes well. Thanks!
